### PR TITLE
es: Translate /locale/es/get-involved/index.md

### PR DESCRIPTION
**Pull request by juliandavidmr**
Translations from English to Spanish, and:

* Remove duplicated of "Community Committee" item.
* Translate "The DEV Community Node.js tag" item.
* Translate "Nodeiflux" item.
* Translate and adjust the "Vietnamese Node.js community".

### CODE REVIEW

**Comment by nschonni** [comment]
Ping @nodejs/nodejs-es

File: locale/es/get-involved/index.md [hunk 1]
```diff
@@ -7,22 +7,21 @@ layout: contribute.hbs
 
 ## Discusión de la comunidad
 
-* La [lista de errores en](https://github.com/nodejs/node/issues) es el lugar para discutir las características del núcleo de Node.js.
+* La [lista de errores](https://github.com/nodejs/node/issues) es el lugar para discutir las características del núcleo de Node.js.
 * Para chatear en tiempo real sobre el desarrollo de Node vaya a `irc.freenode.net` en el canal `#node.js` usando un [cliente de IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) ó conéctese con su navegador al canal usando [WebChat de freenode](https://webchat.freenode.net/#node.js).
```
Comment (Tsuna-mi) [inline locale/es/get-involved/index.md]: Again, another 'o' 🙃 
[Never graphic accent in 'o' as a conjunction](https://www.rae.es/consultas/la-conjuncion-o-siempre-sin-tilde-incluso-entre-cifras)

```suggestion
* Para chatear en tiempo real sobre el desarrollo de Node vaya a `irc.freenode.net` en el canal `#node.js` usando un [cliente de IRC](https://es.wikipedia.org/wiki/Anexo:Clientes_IRC) o conéctese con su navegador al canal usando [WebChat de freenode](https://webchat.freenode.net/#node.js).
```

Comment (juliandavidmr) [reply locale/es/get-involved/index.md]: Entendido, Gracias.

I did not know that conjunction 💯

File: locale/es/get-involved/index.md [hunk 2]
```diff
@@ -7,22 +7,21 @@ layout: contribute.hbs
...
+* [Node Slackers](https://www.nodeslackers.com/) es una comunidad de Slack enfocada en Node.js.
 
 ## Aprendizaje
 
 * La [Documentación oficial de la API](https://nodejs.org/api/) detalla la API de Node.
 * [NodeSchool.io](https://nodeschool.io/) le enseñará conceptos de Node.js de forma interactiva mediante juegos utilizando la línea de comandos.
 * La [etiqueta de Node.js en Stack Overflow](https://stackoverflow.com/questions/tagged/node.js) colecciona nueva información cada día.
-* [The DEV Community Node.js tag](https://dev.to/t/node) is a place to share Node.js projects, articles and tutorials as well as start discussions and ask for feedback on Node.js-related topics. Developers of all skill-levels are welcome to take part.
-* [Nodeiflux](https://discordapp.com/invite/vUsrbjd) is a friendly community of Node.js backend developers supporting each other on Discord.
+* [La etiqueta DEV Community Node.js](https://dev.to/t/node) es un lugar para compartir proyectos, artículos y tutoriales de Node.js, así como para iniciar debates y hacer comentarios sobre temas relacionados con Node.js. Todos los desarrolladores son bienvenidos a ser parte de esta comunidad.
```
Comment (Tsuna-mi) [inline locale/es/get-involved/index.md]: I think is better this way:
```suggestion
* [La etiqueta DEV Community Node.js](https://dev.to/t/node) es un lugar para compartir proyectos, artículos y tutoriales de Node.js, así como para iniciar debates y pedir información sobre temas relacionados con Node.js. Desarrolladores de todos los niveles son bienvenidos a participar.
```

File: locale/es/get-involved/index.md [hunk 3]
```diff
@@ -7,22 +7,21 @@ layout: contribute.hbs
...
+* [Nodeiflux](https://discordapp.com/invite/vUsrbjd) es una comunidad amigable de desarrolladores de back-end de Node.js que se apoyan mutuamente en Discord.
```
Comment (Tsuna-mi) [inline locale/es/get-involved/index.md]:
```suggestion
* [Nodeiflux](https://discordapp.com/invite/vUsrbjd) es una comunidad amigable de desarrolladores back-end de Node.js que se apoyan mutuamente en Discord.
```

File: locale/es/get-involved/index.md [hunk 4]
```diff
@@ -31,4 +30,4 @@ layout: contribute.hbs
 * [Comunidad en Facebook de usuarios Israelí de Node.js](https://www.facebook.com/groups/node.il/)
 * [Grupo de usuarios de Japón](https://nodejs.jp/)
 * [Comunidad en Facebook de Español/Castellano](https://www.facebook.com/groups/node.es/)
-* [Vietnamese Node.js community](https://www.facebook.com/nodejs.vn/)
+* [Comunidad Vietnamita en Facebook](https://www.facebook.com/nodejs.vn/)
```
Comment (Tsuna-mi) [inline locale/es/get-involved/index.md]:
```suggestion
* [Comunidad Vietnamita Node.js](https://www.facebook.com/nodejs.vn/)
```

File: locale/es/get-involved/index.md [hunk 5]
```diff
@@ -31,4 +30,4 @@ layout: contribute.hbs
 * [Comunidad en Facebook de usuarios Israelí de Node.js](https://www.facebook.com/groups/node.il/)
```
Comment (Tsuna-mi) [inline locale/es/get-involved/index.md]: It's plural ;)
```suggestion
* [Comunidad en Facebook de usuarios Israelíes de Node.js](https://www.facebook.com/groups/node.il/)
```

**Review by Tsuna-mi — COMMENTED** [review]
@juliandavidmr before you commit any proposal change or other, I think you need to update the branch with master.
Thanks for the effort

**Comment by juliandavidmr** [comment]
Hi @Tsuna-mi, already been updated with master. Thanks for the review and for teaching me about 'o'.

**Comment by ghost** [comment]
Hi @juliandavidmr：

There're some conflicts to cope with before they are merged.

**Comment by juliandavidmr** [comment]
> Hi @juliandavidmr：
> 
> There're some conflicts to cope with before they are merged.

Hi @MaledongGit 

Fixed. :)

**Comment by ghost** [comment]
Thank you!

**Pull request merged**